sdk-lib/src/androidTest/java/cash/z/ecc/fixture/FakeRustBackend.kt

```diff
@@ -0,0 +1,25 @@
+package cash.z.ecc.fixture
+
+import cash.z.ecc.android.sdk.internal.model.JniBlockMeta
+import cash.z.ecc.android.sdk.jni.RustBackend
+import cash.z.ecc.android.sdk.jni.RustBackendWelding
+import cash.z.ecc.android.sdk.model.BlockHeight
+import cash.z.ecc.android.sdk.model.ZcashNetwork
+
+internal class FakeRustBackend(rustBackend: RustBackend) : RustBackendWelding by rustBackend {
```

Comment:
Do we need `rustBackend` here?  Currently this is not a true fake, but rather a full implementation with just a few methods overridden.

I think it might be preferable to implement all the methods with a `throws` or `error()` call to make it clear in case someone calls this in some unexpected way.